plugins/field-colour/README.md

```diff
@@ -68,13 +98,138 @@ Blockly.defineBlocksWithJsonArray([
       {
         type: 'field_colour',
         name: 'FIELDNAME',
-        colour: '#ffcccc',
+        colour: '#ff4040',
+        colourOptions: [
+          '#ff4040',
+          '#ff8080',
+          '#ffc0c0',
+          '#4040ff',
+          '#8080ff',
+          '#c0c0ff',
+        ],
+        colourTitles: [
+          'dark pink',
+          'pink',
+          'light pink',
+          'dark blue',
+          'blue',
+          'light blue',
+        ],
+        columns: 3,
       },
     ],
   },
 ]);
 ```
 
+The colour constructor takes in the following:
+
+- an optional `value`
+- an optional [validator](#creating_a_colour_validator)
```

Comment:
this (and other links) should use `-` instead of `_`

if you click "view file" in the `...` menu in this PR you can preview the rendered file and see thin links

Reply:
Done. Thanks.